mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -40,7 +40,14 @@ import org.apache.spark.storage.StorageLevel
  * @tparam Item item type
  */
 @Experimental
-class FPGrowthModel[Item: ClassTag](val freqItemsets: RDD[FreqItemset[Item]]) extends Serializable
+class FPGrowthModel[Item: ClassTag](
+    val freqItemsets: RDD[FreqItemset[Item]],
+    val numTransactions: Long) extends Serializable {
+
+  def this(javaFreqItemsets: JavaRDD[FreqItemset[Item]], javaNumTransactions: Long) {
```

Comment:
Let's do this in a separate PR. We need a Java unit test to make sure it works. The `numTransactions` change could be in a separate PR as well (with unit test).